.github/workflows/build.yml

```diff
@@ -1,6 +1,6 @@
 name: continuous integration
 
-on: [push, pull_request]
```

Comment:
Since I'm the original author of this line: I didn't think very hard about what we should choose here.  I copied an example from the github documentation.